README.md

```diff
@@ -43,6 +43,66 @@ open localhost:1313
 - Add the videos you need to [`data/videos.yml`](https://github.com/breadtubetv/breadtubetv/blob/master/data/videos.yml)
 - Add the channels you need to [`data/channels.yml`](https://github.com/breadtubetv/breadtubetv/blob/master/data/channels.yml)
 
+## Script Development
+
+Scripts are being written in Go, this keeps the scripting and operational language the same, provides cross system compatibility, and gives everyone an opportunity to learn a new programming language.
+
+### Installation
+
+#### Installing Go
+
+This is going to be dependent on your system, we recommend following https://golang.org/doc/install
+
+#### Developing
+
+You'll need a copy of the project in the relevent folder.
+
+```
+mkdir -p ~/.go/src/github.com/
```

Comment:
Developers should be able to use `go get github.com/breadtubetv/breadtubetv` which will create the folder structure and clone the project. They'll just have to `cd $GOPATH/src/github.com/breadtubetv/breadtubetv` and checkout the relevant branch.

Reply:
There is currently an issue with the library details not matching the actual endpoint of the project.